pkg/virt-handler/vm.go

```diff
@@ -694,7 +694,6 @@ func (c *VirtualMachineController) migrationSourceUpdateVMIStatus(origVMI *v1.Vi
 		vmi.Status.NodeName = migrationHost
 		// clean the evacuation node name since have already migrated to a new node
 		vmi.Status.EvacuationNodeName = ""
-		vmi.Status.MigrationState.Completed = true
```

Comment:
Can you also remove the comment above, which is now misleading?

```
		// this is the migration ACK.
		// At this point we know that the migration has completed and that
		// the target node has seen the domain event.
```

Reply:
The comment is still relevant, when that bit of code is executed the migration has actually ended.